core/node/rpc/sync/client/local.go

```diff
@@ -100,7 +100,11 @@ func (s *localSyncer) OnUpdate(r *StreamAndCookie) {
 	default:
 		log := dlog.FromCtx(s.syncStreamCtx)
 		log.Error("Cancel client sync operation - client buffer full", "syncId", s.globalSyncOpID)
-		s.cancelGlobalSyncOp()
+
+		err := RiverError(Err_BUFFER_FULL, "Client sync subscription message channel is full").
+			Tag("syncOpId", s.globalSyncOpID).
```

Comment:
using same tag name allows to group messages
```suggestion
			Tag("syncId", s.globalSyncOpID).
```